<?xml version="1.0" encoding="UTF-8"?>
<testsuites xmlns:a="http://www.gallio.org/" xmlns:functx="http://www.functx.com">
   <testsuite skipped="0" failures="1" errors="0" time="0.8568407" tests="3"
              name="UnitTests.MainClassTest">
      <testcase time="0.41357679999999997" name="TestFailure"
                classname="UnitTests.MainClassTest">
         <failure message="">   at NUnit.Framework.Assert.That(Object actual, Constraint constraint, String message, Object[] args)
   at NUnit.Framework.Assert.AreEqual(Int32 expected, Int32 actual, String message, Object[] args)
   at NUnit.Framework.Assert.AreEqual(Int32 expected, Int32 actual)
   at UnitTests.MainClassTest.TestFailure() in D:\Projets\TU\NUnit\GallioFailure\UnitTests.cs:ligne 24</failure>
      </testcase>
      <testcase time="0.093309599999999993" name="TestMethodUpdateValue"
                classname="UnitTests.MainClassTest"/>
      <testcase time="0.0477094" name="TestPropertyValue" classname="UnitTests.MainClassTest"/>
   </testsuite>
</testsuites>